app/models/value_set.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ValueSet
+  attr_reader :codes
+
+  def initialize(file)
+    @codes = FHIR.from_contents(File.read(file)).compose.include.map do |compose|
+      compose.concept.map do |concept|
+        FHIR::Coding.new(system: compose.system, code: concept.code, display: concept.display)
+      end
+    end.flatten
```

Comment:
Rather than `.map.flatten` you can use `.flat_map`.

Reply:
resolved